src/app/containers/RelatedTopics/index.test.jsx

```diff
@@ -118,22 +119,8 @@ describe('Expected use', () => {
     );
   });
 
-  it('should return null on story pages when the topicsTags toggle is disabled on CPS', () => {
-    const { container } = render(
-      <WithContexts cpsEnabled={false} pageType={STORY_PAGE}>
-        <RelatedTopics
-          topics={[
-            { topicName: 'topic1', topicId: '1' },
-            { topicName: 'topic2', topicId: '2' },
-            { topicName: 'topic3', topicId: '3' },
-          ]}
-        />
-      </WithContexts>,
-    );
-    expect(container.firstChild).toBeNull();
-  });
-
-  it('should return null on article pages when the topicsTags toggle is disabled on Optimo', () => {
+  it('should return null when on the live environment', () => {
```

Comment:
As discussed above I'd add a test to cover when `process` is not defined at all.